frontend/src/canvas/CanvasContainer.js

```diff
@@ -48,6 +49,7 @@ const CanvasContainer = (props) => {
   };
 
   const handlePointerMove = (e) => {
+    if (props.nftMintingMode) return;
```

Comment:
Can you add another check so we can move the canvas again if `nftSelected` is true

Reply:
okay